ubuntu/14.04/standard/Dockerfile

```diff
@@ -118,7 +118,8 @@ ENV dotnet=/usr/bin/dotnet
 # Install AzCopy (depends on .NET Core)
 RUN apt-get update \
  && apt-get install -y --no-install-recommends azcopy \
- && rm -rf /var/lib/apt/lists/*
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get purge -y azcopy 
```

Comment:
This isn't what I meant. I was suggesting that you encompass the install of AzCopy, the usage of azcopy to download the hosted tool cache, and the uninstall of azcopy into a single RUN statement so it doesn't get baked into one of the image layers. As it stands, the subsequent usage of azcopy will not find the command.

Reply:
Something else to be aware of is that apt-get install azcopy will add the .NET Core 2.0.0 runtime into the image (and apt-get purge will not remove it by default). This might be an ok thing to do (we probably should be installing a variety of .NET Core runtime versions) but we should be aware that it's adding to an already huge image size.

Reply:
Probably the simplest thing to do for now would be to leave azcopy in the image. Someone in some VSTS task somewhere will find it useful :)

Reply:
Got it.  Yeah I realized I was removing it in the wrong place.  I'll change it back.